### DISCUSSION TITLE
Missing terrain in Brazillian airport Curitiba (SBCT)

### DISCUSSION THREAD

**Discussion by rquintelarj** [kubilus1/autoortho#200] [General]
Hello!

Using AutoOrtho, nearby and at Curitiba airport (SBCT - XP11 and 12), the scenery disappears, becoming like the image.

Does anyone know how to fix?

![X-Plane Screenshot 2023 06 05 - 15 19 11 40](https://github.com/kubilus1/autoortho/assets/135556412/5279cd17-d7cc-4494-8081-b7e824f35be3)

**Comment by rquintelarj** [comment]
Same problem in 0.5.0 version.

**Comment by zSan1** [comment]
Same problem

**Comment by kubilus1** [comment]
Yeah, I can replicate this as well.  Probably an issue with the scenery generation side.

**Comment by kubilus1** [comment]
I tried re-generating the tile and it still looked like this.  Might be an issue with Ortho4XP.

**Comment by rquintelarj** [comment]
[-26-050.zip](https://github.com/kubilus1/autoortho/files/11859917/-26-050.zip)


A temporary fix.


Unpack and Overwrite this file in Custom Scenery\z_autoortho\scenery\z_ao_sa\Earth nav data\-30-050

**Reply by kubilus1** [reply]
Thanks!  Can you share the Ortho4XP.cfg you used for that, so I can incorporate whatever makes this work.

**Reply by rquintelarj** [reply]
I acquired this scenario file from this page:

https://axscenery.com/sbct-5-0-afonso-pena/

Its free.

**Comment by xpln1** [comment]
Hi! I had the same problem with tile +31+035 and I apparently fixed it by regenerating it. I replaced the earth nav data in ./yAutoOrtho_Overlays and ./z_eur_02/, where I also added the terrain files and then the textures in ./z_autoortho.
Hope this helps!
[Ortho4XP_+31+035.txt](https://github.com/kubilus1/autoortho/files/11899283/Ortho4XP_%2B31%2B035.txt)

**Reply by kubilus1** [reply]
Nice!  I'll try those setting with this.  BTW this is the project that generates the scenery for this project.  https://github.com/kubilus1/autoortho-scenery

**Reply by kubilus1** [reply]
Do you know which of those settings allows it to work or not?

**Reply by xpln1** [reply]
Oddly enough it worked with those configs as well, however I'm not 100% sure some files from the previous installations didn't remain there. It may be a problem with the script, I don't know how to run it for a specific tile and barely understand it. I don't know if it has anything to do with it, but I select all the checkboxes under _Batch build _tiles_ in Ortho4XP when I do the downloading.

**Reply by kubilus1** [reply]
I show the same issues with those settings.  I think this is an ortho4xp issue, if someone wants to report there.

**Reply by Tiggs62** [reply]
I got around this by adding HD Mesh - same for Iceland, which shows up as flat

**Comment by d41k4n** [comment]
This was a Ortho4XP/OSM coastline issue, which I hopefully fixed. See issue https://github.com/kubilus1/autoortho/issues/239 for details.

**Comment by kubilus1** [comment]
Latest scenery release resolves this.
